### PR TITLE
NNBD migration without actually migrating

### PR DESCRIPTION
**Pull request by passsy**
Changes from #122 except that all nnbd changes are commented out, making #122 easier to review.

The most significant change is in the tests with the removal of mockito

### CODE REVIEW

**Comment by passsy** [comment]
Merging this because the tests on master are currently failing and I continue working on #126

File: lib/src/common/widgets/dismissible_page_route.dart [hunk 1]
```diff
@@ -325,32 +317,28 @@ class _DownGestureController<T> {
     }
 
     if (animateForward) {
-      final int droppedPageForwardAnimationTime = min(
-        lerpDouble(
-                _kMaxDroppedSwipePageForwardAnimationTime, 0, controller.value)
-            .floor(),
-        _kMaxPageBackAnimationTime,
-      );
+      final a = lerpDouble(
```
Comment (atoka93) [inline lib/src/common/widgets/dismissible_page_route.dart:320]: What about naming this better?

File: lib/src/common/widgets/dismissible_page_route.dart [hunk 2]
```diff
@@ -325,32 +317,28 @@ class _DownGestureController<T> {
...
       navigator.pop();
-      if (onPagePopped != null) {
-        onPagePopped();
-      }
+      onPagePopped?.call();
       if (controller.isAnimating) {
-        final int droppedPageBackAnimationTime = lerpDouble(
-                0, _kMaxDroppedSwipePageForwardAnimationTime, controller.value)
-            .floor();
+        final a = lerpDouble(
```
Comment (atoka93) [inline lib/src/common/widgets/dismissible_page_route.dart:331]: What about naming this better?

File: lib/src/feedback/components/input_component.dart [hunk 3]
```diff
@@ -102,6 +100,7 @@ class _InputComponentState extends State<InputComponent> {
             errorMaxLines: 2,
           ),
           maxLength: _maxInputLength,
+          // ignore: deprecated_member_use
```
Comment (atoka93) [inline lib/src/feedback/components/input_component.dart:103]: What's the alternative here?

**Pull request merged**